test/wallet-rpc-test.js

```diff
@@ -288,6 +289,53 @@ describe('Wallet RPC Methods', function() {
     });
   });
 
+  it('should rpc gettransaction a sent TX', async () => {
```

Comment:
It will be better to group new RPC tests with `describe`. `describe('gettransaction', ...`

Reply:
thanks, fixed.